tests/unit/test_resources.py

```diff
@@ -60,3 +62,20 @@ def test_cli(tmpdir):
         raise AssertionError(
             f'{result.stderr}'
         )
+
+
+def test_extract_tutorials(monkeypatch, tmp_path, caplog):
+    test_dest = tmp_path/ 'destination'
+    class MockGlobalConfig:
+        def get(*args):
+            return [str(test_dest)]
+
+    monkeypatch.setattr(GlobalConfig, '_DEFAULT', MockGlobalConfig)
```

Comment:
You could also use the `mock_glbl_cfg` fixture
```
mock_glbl_cfg(
    'cylc.flow.resources.glbl_cfg',
    f'''
    [install]
        source dirs = {test_dest}
     '''
)
```

Reply:
Thanks - I thought that such a thing existed, but I was clearly not thinking straight at the time I wrote this.